CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # bedrock ChangeLog
 
+## 4.4.2 - 2021-10-xx
+
+### Fixed
+- Signal handlers that prematurely terminated the primary or its workers
+  have been refactored to allow an orderly exit. The primary and worker
+  internal messaging and exit process has been cleaned up and simplified to
+  help ensure more consistent outcomes and eliminate a number of bugs or
+  unexpected states. Additionally, some unused IPC messages were removed.
+  These changes should not have affected exiting applications except in those
```

Comment:
Sounds odd to be talking about current changes in past tense.
```suggestion
  These changes should not affect exiting applications except in those
```

Reply:
Well, I wrote it that way because it was my intention not to affect -- but there could be bugs. I'll reword.